middlewares/validateToken.js

```diff
@@ -0,0 +1,34 @@
+import jwt from 'jsonwebtoken';
+import models from '../models/index';
+
+const { user: UserModel } = models;
+
+const Auth = {
+  /**
+    * Verify Token
+    * @param {object} req
+    * @param {object} res
+    * @param {object} next
+    * @returns {object|void} response object
+    */
+  async verifyToken(req, res, next) {
+    const token = req.headers['x-access-token'];
+    if (!token) {
+      return res.status(401).send({ error: 'No token was provided' });
+    }
+    try {
+      const decoded = await jwt.verify(token, process.env.SECRETKEY);
+      const userInfo = await UserModel.findOne({ where: { id: decoded.id } });
```

Comment:
Am not sure if you should be calling the modal in the middleware? but i think we shouldn't be doing so.